### DISCUSSION TITLE
yiwen-blog-website/docs/web/intro

### DISCUSSION THREAD

**Discussion by giscus** [Hao-yiwen/yiwen-blog-website#2] [Announcements]
# yiwen-blog-website/docs/web/intro

java学习框架

http://localhost:3000/yiwen-blog-website/docs/web/intro

**Comment by Hao-yiwen** [comment]
评论测试！！！

**Comment by Hao-yiwen** [comment]
归纳一套涉及服务端/web端/小程序/native端的最佳实践，独立并且有效支撑各类业务需求。
